includes/class-ssp-frontend.php

```diff
@@ -294,13 +308,77 @@ public function get_episode_download_link( $episode_id, $referrer = '' ) {
 		return apply_filters( 'ssp_episode_download_link', esc_url( $link ), $episode_id, $file );
 	}
 
+	/**
```

Comment:
New method to obtain album art, used by the `media_player()` method. This will iteratively look for the most appropriate image, starting from the featured image and working down to the default feed image, using the first one it encounters (in order of relevance) that is square.